addons/misc/XEH_preInit.sqf

```diff
@@ -57,6 +57,86 @@ PREP_RECOMPILE_END;
     true
 ] call CBA_Settings_fnc_init;
 
+//Adjust armband sling left arm (x, y, z)
+[
+    QGVAR(armbandSlingLeftArm),
+    "EDITBOX",
+    [LLSTRING(SETTING_ArmbandSlingLeftArmPos_Display), LLSTRING(SETTING_ArmbandSlingLeftArmPos_Desc)],
+    [CBA_SETTINGS_CAT, LSTRING(SubCategory_Armband)],
+    "0.2, -0.39, -0.2",
```

Comment:
both arms default position looks like nonsense, during my tests they were located on my elbows and below them, legs are kinda "ok" tho who the fuck is placing armband on their thigh (or knee, its hard to say)

Reply:
Due to Armas restrictions on which bones allow you to attach an item I had to use lowerArm instead of upperArm. That leads to this position, saidly nothing you can do about it. I think the right arm looks fine but the other one, yea that looks a bit sloppy but best I can do, but do to different uniforms having other arm width you can adjust it to your liking in the settings.

Reply:
if it looks like that, then why adding these as action?

Reply:
I think it looks okay, you have to take into account tho that they look different depending on uniform. I work for the philosophy "Provide as much freedom for the user as possible" and even if it's just one person using it, it was worth it. 
Although due to that you can configure it in CBA you can make it how you like it. BTW a short trip to the real world: Medics usually wear the red crosses a bit more hidden then on the upper arm to not make them stand out and make a easy target for enemies. (I know Geneva but terrorist ignore those)  So they tend to wear them either on the lower arm or on knees or legs. So from that POV that position is kinda realistic in some way. But I get your point I'd love to make them attach to the upper arm as well. 😭